test/package.json

```diff
@@ -6,11 +6,14 @@
     "@types/reflect-metadata": "^0.0.5",
     "@types/selenium-webdriver": "^3.0.7",
     "ava": "^0.23.0",
+    "chai": "^4.1.2",
```

Comment:
Revert changes to this file. These libraries are not required for this PR.

Reply:
Removed.